File: day-5/search-key.c
#include <stdio.h>
#include <stdlib.h>
int main(void){
    int a[1001],i,limit;
    printf("Enter your Array limit :");
    scanf("%d",&limit);
    printf("Enter your Values :");
    for ( i = 0; i < limit; i++)
    {
        scanf("%d",&a[i]);

    }
    int searchkey,flag=0;
    printf("Enter the value you wanted to search :");
    scanf("%d",&searchkey);
    for ( i = 0; i < limit; i++)
    {
        if (searchkey ==  a[i])
        {
            printf("Item has been found at :%d\n",i+1);
            flag=1;
            break;

        }
        
        
    }
    if (flag==0)
    {
        printf("Value not found\n");
    }
    
    
    














    return EXIT_SUCCESS;
}

File: day-7/SearchKey.java
import java.util.Scanner;

public class SearchKey {
    public static void main(String[] args) {
        System.out.print("Enter your Array limit");
        try (Scanner s = new Scanner(System.in)) {
            int limit=s.nextInt();
            
            System.out.print("Enter your Array :");
            int[] a=new int[limit]; 
            for (int i = 0; i < limit; i++) {
                a[i]=s.nextInt();
            }

            System.out.print("Enter value for search");
            int key=s.nextInt();
            
            System.out.println(a[2]);
            for (int i = 0; i < limit;i++){
                if (key == a[i]) {
                    
                    System.out.print("value found at "+(i+1));
                    break;

                    
                }
                else{
                    System.out.print("Value not found");
                    
                    
                }
                
                
            }
            
        }
    }
    
}
